assets/js/player.js

```diff
@@ -593,20 +598,20 @@ addEventListener('keydown', function (e) {
         return;
     }
     // See https://github.com/ctd1500/videojs-hotkeys/blob/bb4a158b2e214ccab87c2e7b95f42bc45c6bfd87/videojs.hotkeys.js#L310-L313
-    const isPlayerFocused = false
-        || e.target === document.querySelector('.video-js')
-        || e.target === document.querySelector('.vjs-tech')
-        || e.target === document.querySelector('.iframeblocker')
-        || e.target === document.querySelector('.vjs-control-bar')
+    const isPlayerFocused = false ||
+        e.target === document.querySelector('.video-js') ||
+        e.target === document.querySelector('.vjs-tech') ||
+        e.target === document.querySelector('.iframeblocker') ||
+        e.target === document.querySelector('.vjs-control-bar')
```

Comment:
I have mixed feelings on this one... But I guess this isn't an isolated config option?

Reply:
I added a rule for this to match the stylings of other files, I can remove this rule or inverse it if preferred